android/src/org/mozilla/firefox/vpn/InAppPurchase.kt

```diff
@@ -344,8 +346,25 @@ class InAppPurchase private constructor(ctx: Context) :
         formatter.maximumFractionDigits = 2
         formatter.currency = Currency.getInstance(details.priceCurrencyCode)
         val monthlyPriceString = formatter.format(monthlyPrice)
+
+        // freeTrialPeriod is a ISO 8601 duration i.e P7D == 7 days
+        val trialDays = if (monthCount == 12 && VPNUtils.isDevMode) {
+            // TODO: This is just some debugging stuff
+            // We need to be on prod so, for testing let's
+            // mock the PlayStore response for 12month if we're in isDevMode
+            // so QA can test when we go live
+            val duration = Duration.parse("P7D")
+            duration.toDays().toInt()
```

Comment:
I understand this is for QA to use for testing, but If a user happens to have dev options enabled and no free trial available for the 12mo option, and this option still appears, could this be misleading? Or do we just assume a user shouldn't have dev options enabled?